src/scenes/success/success.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styles from "./success.module.scss";
+import { CircleBig, PositionCircleBigInfo } from "../../components/circleBig";
+import {
+  CircleSmall,
+  PositionCircleSmallInfo,
+} from "../../components/circleSmall";
+import { useDispatch } from "react-redux";
+import {
+  updateCurrentScene,
+  updateStartAnimation,
+} from "../../store/appControl/appControlSlice";
+
+export function Success() {
+  const dispatch = useDispatch();
+
+  const circleBigPositions: PositionCircleBigInfo[] = [
```

Comment:
it' better to define constants outside of the component body

Reply:
fixed